src/components/sidebar/sidebar.js

```diff
@@ -45,15 +48,27 @@ export const Sidebar=()=>{
             <div className="product-category flex flex-direction-col p-1">
                 <h2>catrgory</h2>
                 <div className="py-0-5">
-                    <input type="checkbox" onChange={()=>dispatch({type:"ios"})} /><span>IOS</span>
+                    <input checked={state.category.grocery} type="checkbox" onChange={()=>dispatch({type:"grocery"})} /><span>Grocery</span>
+                </div>
+                <div>
+                    <input checked={state.category.mobile} type="checkbox" onChange={()=>dispatch({type:"mobile"})} /><span>Mobile</span>
+                </div>
+                <div>
+                    <input checked={state.category.home} type="checkbox" onChange={()=>dispatch({type:"home"})} /><span>Home</span>
+                </div>
+                <div>
+                    <input checked={state.category.electronics} type="checkbox" onChange={()=>dispatch({type:"electronics"})} /><span>Electronics</span>
+                </div>
+                <div>
+                    <input checked={state.category.fashion} type="checkbox" onChange={()=>dispatch({type:"fashion"})} /><span>Fashion</span>
```

Comment:
you can use destructuring over here rather than writing "state.category" again n again

Reply:
yes yes got it